code/modules/shuttle/ripple.dm

```diff
@@ -0,0 +1,15 @@
+/obj/effect/ripple
+	name = "bluespace ripple"
+	desc = "Something is coming through bluespace, you can see the \
+		visual disturbances. It's probably best not to be on top of these \
+		when whatever is tunneling comes through."
+	icon = 'icons/effects/effects.dmi'
+	icon_state = "medi_holo"
```

Comment:
They're placeholder animations anyway, although I do like the circle that it ends on.